tools/small_read_size_histograms/size_histogram.xml

```diff
@@ -1,15 +1,14 @@
-<tool id="Size_histogram" name="Generate size histograms from alignment files" version="0.9.9">
-    <description>from sRbowtie aligment</description>
+<tool id="artbio_size_histogram" name="Generate read size histograms" version="1.0.0">
+    <description>from alignment files</description>
     <requirements>
-        <requirement type="package" version="1.1.2">bowtie</requirement>
-        <requirement type="package" version="0.8.3">pysam</requirement>
-        <requirement type="package" version="3.1.2">R</requirement>
-        <requirement type="package" version="2.14">biocbasics</requirement>
-        <requirement type="package" version="1.9">numpy</requirement>
-        <requirement type="package" version="2.0.0=r3.2.2_0a">r-gridextra</requirement>
-        <requirement type="package" version="0.6_26=r3.2.2_2a">r-latticeextra</requirement>
+        <requirement type="package" version="1.2.0=py27_0">bowtie</requirement>
+        <requirement type="package" version="0.11.2.1=py27_0">pysam</requirement>
```

Comment:
Yeah. I still have the shift to py3 in mind. But I want to clear off smRtools module before. I'll change to py3 when I remove the smRtools dependency from size_histogram.py.
You may do the same for your current dev, since you are also squeezing smRtools ;-)